package.json

```diff
@@ -109,15 +109,25 @@
           "default": [],
           "description": "These custom associations will override the folder icon associations defined by default."
         },
-        "vsicons.associations.fileDefault": {
+        "vsicons.associations.fileDefault.file": {
           "type": "object",
           "default": null,
-          "description": "This setting will let you change default file icons (dark & light theme)."
+          "description": "This setting will let you change default dark file icon."
```

Comment:
This contribution point is only suited to change the `default dark file icon`, thus the use of the singular. Note that there is a specific contribution point for every default icon possible.